src/index.html

```diff
@@ -3,14 +3,18 @@
 ---
 
 <header class="hero">
-  <h1 class="hero__title">
-    British Columbia Made
-    <small>Since 1987</small>
-  </h1>
-
-  <p class="hero__tagline">Premium Quality Industrial Safety Garments</p>
-
-  <a class="button button--accent" href="/products">Our Products</a>
+  <div class="title__container">
+      <h1 class="hero__title">
+         BRITISH COLUMBIA MADE
```

Comment:
use `text-transform: uppercase` to make things uppercase, write your content like it would be without styling.